brightlayer-ui/brightlayer-ui-cli
# Remove Expo from the CLI

Fixes #114, BLUI-2066


#### Changes proposed in this Pull Request:
- Drop support for EXPO
- Updated Read Me


#### Screenshots:

Whether you are running `blui new` or `blui new react-native --cli=expo --name=asdf --language=js --lint --prettier`, they all work. It does not ask you which CLI you want to use anymore and spins up a healthy React Native project by default.

![image](https://user-images.githubusercontent.com/8997218/159978287-f8528434-7b51-4204-bfb0-5cbd2221d4f5.png)



#### To Test:
- follow the instruction in the readme 
-  Try testing commands like `blui new react-native --cli=expo`. You will see that the command prompt reacts as if there's no `cli` flag at all.
